patch_python_extension.py:
  - url: https://ae5-vscode.s3.amazonaws.com/patch_python_extension.py
    sha256: 9ea080b07fac8135e2aa95f0cf6290aca07a1f981d7c955d96cb0706882a6778
   
code-server.tar.gz:
  - url: https://ae5-vscode.s3.amazonaws.com/code-server-3.9.3-linux-amd64.tar.gz
    sha256: eba42eaf868c2144795b1ac54929e3b252ae35403bf8553b3412a5ac4f365a41

extensions:
  - url: https://ae5-vscode.s3.amazonaws.com/ae5-session-0.3.1.vsix
    sha256: 412264942db710e52506974ca9e4c99dd681be3fb6707fb55a4cfabf1f941167

  - url: https://ae5-vscode.s3.amazonaws.com/ms-toolsai.jupyter-2021.3.0.vsix
    sha256: 33843b2af436d999ee08f62c2719d12718a2e9d8bee6068c14d43fe04d20b083

  - url: https://ae5-vscode.s3.amazonaws.com/ms-python-2021.5.926500501.vsix
    sha256: 637a1de6d5494eb56922c7498b6d4cdaafd68a56160382c0d7e14d0fc20feb42
    post_install:
      - "/opt/continuum/anaconda/envs/lab_launch/bin/python patch_python_extension.py /opt/continuum/scripts/skeletons/user/home/.vscode/extensions/ms-python.python-2021.5.926500501 --preparing-env"

